lib/rules/syntaxes/slot-attribute.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * @author Yosuke Ota
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+module.exports = {
+  deprecated: '2.6.0',
+  createTemplateBodyVisitor (context) {
+    const sourceCode = context.getSourceCode()
+
+    /**
+     * Checks whether the given node can convert to the `v-slot`.
+     * @param {VAttribute} slotAttr node of `slot`
+     * @returns {boolean} `true` if the given node can convert to the `v-slot`
+     */
+    function canConvertFromSlotToVSlot (slotAttr) {
+      if (!slotAttr.value) {
+        return true
+      }
+      const slotName = slotAttr.value.value
+      // If non-Latin characters are included it can not be converted.
+      return !/[^a-z]/i.test(slotName)
+    }
+
+    /**
+     * Checks whether the given node can convert to the `v-slot`.
+     * @param {VAttribute} slotAttr node of `v-bind:slot`
+     * @returns {boolean} `true` if the given node can convert to the `v-slot`
+     */
+    function canConvertFromVBindSlotToVSlot (slotAttr) {
+      if (!slotAttr.value) {
+        return true
+      }
+
+      if (!slotAttr.value.expression) {
+        // parse error or empty expression
+        return false
+      }
+      const slotName = sourceCode.getText(slotAttr.value.expression).trim()
+      // If space is included it can not be converted.
+      return !/\s/.test(slotName)
```

Comment:
Would you make this check stricter a bit? It's good if the same check as `canConvertFromSlotToVSlot` function. For example, it should reject `a>b?c:d` or something like.

```suggestion
      return !/[^a-z]/i.test(slotName)
```

Reply:
Thank you for telling me that issue!
I fixed that.